### PR TITLE
Feature/376 support of postgresql 10 when init the application

### LINKED ISSUE #376: Flyway - Suppport of PostgreSQL 10  when init the application
**Is your feature request related to a problem? Please describe.**
If we want to init the application with PostgreSQL10 it crash because their is some difference between MySQL and PostgreSQL

**Describe the solution you'd like**
Have the possibility to switch between one or another configuration using the properties.

**Describe alternatives you've considered**

If possible provide visual sample and code impacts

**NB**: Specify label(s) in right panel to ease requests organization

### PR DESCRIPTION
**Pull request by Juyoo**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] Make sure you are making a pull request against the **dev** (left side). Also you should start *your branch* off *dev*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Make sure that you've add licence text on every new file you are adding to repository.

### Description

*Please describe your pull request.
If possible provide visual elements to illustrate your feature and*

* Add support for postegresql database

Thank you for your contribution ! :tropical_drink:

Closes #376

### CODE REVIEW

**Comment by Juyoo** [comment]
It's working with mysql and postgre.
You have just to change the properties : spring.flyway.locations

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ server.port: 8080
 spring:
   ### Database ###
   datasource:
-    url: "jdbc:h2:file:./h2/suricate;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE"
+    url: "jdbc:h2:file:./h2/suricate;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=PostgreSQL"
```
Comment (yodamad) [inline src/main/resources/application.yml]: Will it work with MySQL ??

Comment (Juyoo) [reply src/main/resources/application.yml]: Because i have set Postgre as default migration folder below.

Comment (yodamad) [reply src/main/resources/application.yml]: Can we use spring profiles to have both available ?

**Pull request merged**